Apps/Src/BlinkLights.c

```diff
@@ -0,0 +1,93 @@
+#include "Tasks.h"
+#include "Lights.h"
+
+void Task_BlinkLight(void* p_arg){
+    //get necessary state pointers
+    car_state_t *car_state = (car_state_t*) p_arg;
+    blinker_states_t *blinkerStates = &(car_state -> BlinkerStates);
+    light_switches_t *storedLightSwitches = &(car_state -> SwitchStates.lightSwitches);
+    CPU_TS ts;
+    OS_ERR err;
```

Comment:
Are there any safety issues if we get an OS error, or does the controls system recover fine?

Reply:
@ClarkPoon I do not believe we currently have OS Error fault handling, does BPS address this in any particular manner? I believe the plan moving forward was to build fault handling once we've got the core application threads completed.

Reply:
The BPS handles OS faults by shutting down the car using [this function.](https://github.com/lhr-solar/BPS/blob/c0aabfb1390afedae6c89bb5627b340984ad060a/Apps/Src/Tasks.c#L71) I'm not sure what the expected behavior should be for controls (eg. shut down the motor or try to recover). 

Fault handling can sometimes be helpful for debugging the application/thread layer, since the OS errors I have seen were mainly caused by bugs in the BPS's application layer (eg. from forgetting to initialize something or forgetting to release a mutex)

Reply:
I like this idea. I think ideally, each individual application tries to recover from any faults, and if not, we resort to a fault handler that shuts down the system. I'll make an issue ticket/note for this.